### DISCUSSION TITLE
better display for long list of chips (authors or tags)

### DISCUSSION THREAD

**Discussion by uniquename** [gotson/komga#1585] [Ideas]
### Is your feature request related to a problem? Please describe.
When there are a lot of writers or pencilers for a book, they are displayed with a  horizontal scrolling on the book detail page. On touch- and smaller screeens its a bit fiddly to scroll and to click on them. Plus you can not see them all at a glance. 

### Describe the solution you'd like
Couldn't they be displayed without horizontal scrolling and in new lines instead? 

...fantastic app by the way!-)

**Comment by gotson** [comment]
With a long list it would take the whole screen, not very convenient.

**Comment by uniquename** [comment]
Maybe opening and closing it, like the description?

**Reply by Blazeflack** [reply]
I like the idea of being able to click to show it all, instead of horizontal scrolling.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.26.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.26.0))
